### PR TITLE
fix #35955 ads.businessmixer.net

### LINKED ISSUE #35955: ads.businessmixer.net
### Issue URL (Incorrect Blocking)
[http://ads.businessmixer.net/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=http%3A%2F%2Fads.businessmixer.net%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>
<details>
  <summary>Screenshot 3</summary>

  Screenshot 3
</details>

### System configuration

Information | value
--- | ---
Platform: | Ext
AdGuard version: | 3.1.3
Browser: | Chrome
Filters: | AdGuard Base

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/35955

### CODE REVIEW

File: RussianFilter/sections/whitelist.txt [hunk 1]
```diff
@@ -36,6 +36,8 @@ beko.co.uk#@#.topbanner
 @@||vidalak.com/video.css$domain=multplayer.ru
 @@||vidalak.com/api/vpaid^$domain=multplayer.ru
 @@||vidalak.com/player/vpaid^$domain=multplayer.ru
+! https://github.com/AdguardTeam/AdguardFilters/issues/35955
+@@||ads.businessmixer.net^$domain=ads.businessmixer.net
```
Comment (BlazDT) [inline RussianFilter/sections/whitelist.txt]: Please also add rule for iOS/Safari/content blocker. Site will not open even with that rule.

**Review by BlazDT — CHANGES_REQUESTED** [review]

File: RussianFilter/sections/whitelist.txt (hunk 1 above)
Comment (Alex-302) [reply RussianFilter/sections/whitelist.txt]: @BlazDT why? It works in Safari/iOS.

**Review by Alex-302 — APPROVED** [review]

File: RussianFilter/sections/whitelist.txt (hunk 1 above)
Comment (Alex-302) [reply RussianFilter/sections/whitelist.txt]: in any case this site has 0 visitors.

Comment (BlazDT) [reply RussianFilter/sections/whitelist.txt]: Normally iOS blocks it as it contains ads in its url.

**Pull request merged**